web/src/components/Change.res

```diff
@@ -264,16 +264,70 @@ module RowItem = {
     </tr>
 }
 
+module Pagination = {
+  @react.component @module("@patternfly/react-core")
+  external make: (
+    ~children: 'children=?,
+    ~className: string=?,
+    ~defaultToFullPage: bool=?,
+    ~dropDirection: @string
+    [
+      | @as("up") #Up
+      | @as("down") #Down
+    ]=?,
+    ~firstPage: int=?,
+    ~isCompact: bool=?,
+    ~isDisabled: bool=?,
+    ~isStatic: bool=?,
+    ~isSticky: bool=?,
+    ~itemCount: int,
+    ~itemsEnd: int=?,
+    ~itemsStart: int=?,
+    ~offset: int=?,
+    ~onPerPageSelect: (ReactEvent.Mouse.t, int, int, int, int) => unit=?,
+    ~onSetPage: (ReactEvent.Mouse.t, int, int, int, int) => unit=?,
+    ~onNextClick: (ReactEvent.Mouse.t, int) => unit=?,
+    ~onPreviousClick: (ReactEvent.Mouse.t, int) => unit=?,
+    ~onPageInput: (ReactEvent.Mouse.t, int) => unit=?,
+    ~page: int=?,
+    ~perPage: int=?,
+    ~variant: @string
+    [
+      | @as("top") #Top
+      | @as("bottom") #Bottom
+      | @as("PaginationVariant") #PaginationVariant
+    ]=?,
+    ~widgetId: string=?,
+  ) => React.element = "Pagination"
+}
```

Comment:
The binding is now part of re-patternfly-3.0.2, could you please update the package.json (and run npm install after) instead?
```suggestion
```

Reply:
Done